TestVectors/dafny/TestVectors/src/EsdkTestVectors.dfy

```diff
@@ -151,6 +151,21 @@ module {:options "-functionSyntax:4"} EsdkTestVectors {
         description: string,
         decryptionMethod: DecryptionMethod
       )
+    | PositiveV1DecryptTestVector(
```

Comment:
Where does "V1" come from? What is "V1" referring to?

Reply:
V1 is the manifest version

Reply:
It might be helpful to call it `PositiveV1ManifestDecryptTestVector` but I won't block on that since you can find it from other context

Reply:
`PositiveV1OrV2ManifestDecryptTestVector`